aten/src/ATen/native/quantized/cpu/qreduction.cpp

```diff
@@ -8,7 +8,7 @@
 namespace at {
 namespace native {
 #ifdef USE_PYTORCH_QNNPACK
-Tensor qnnpack_mean(const Tensor& input, IntArrayRef dim) {
+Tensor qnnpack_mean(const Tensor& input, IntArrayRef dim, bool keepdim) {
```

Comment:
Can you also add `TORCH_CHECK` assert that checks for `dim[0] == 2 and dim[1] == 3`. I know current code path gets executed only in that case but it will be good to protect here.

Reply:
sure

Reply:
actually, this check is already there